pyi.Dockerfile

```diff
@@ -343,10 +331,9 @@ COPY ./scripts/list_needed_libs.sh ./scripts/list_needed_libs.sh
 # hadolint ignore=SC2046,SC2086
 RUN set -e; \
     if [ $(uname -m) != "aarch64" ]; then \
-        source scl_source enable devtoolset-8 llvm-toolset-7 ; \
```

Comment:
Not needed here anymore - because we needed devtoolset for BCC and now it's a separate layer so we don't require it anymore?

Reply:
That's right. We no longer need those toolsets around for staticx to compile gprofiler.